domains/pallets/messenger/src/lib.rs

```diff
@@ -215,11 +215,7 @@ mod pallet {
         /// Hash type of MMR
         type MmrHash: Parameter + Member + Default + Clone;
         /// MMR proof verifier
-        type MmrProofVerifier: MmrProofVerifier<
-            Self::MmrHash,
-            BlockNumberFor<Self>,
-            StateRootOf<Self>,
-        >;
+        type MmrProofVerifier: MmrProofVerifier<Self::MmrHash, BlockNumberFor<Self>, StateRootOf<Self>>;
```

Comment:
It looks like this commit contains some formatting and some manual cleanups, please try to keep them separate.